src/platform/testing/e2e/cypress/plugins/index.js

```diff
@@ -1,8 +1,8 @@
 const fs = require('fs');
 const webpackPreprocessor = require('@cypress/webpack-preprocessor');
-const table = require('table').table;
-const DefinePlugin = require('webpack').DefinePlugin;
-const ProvidePlugin = require('webpack').ProvidePlugin;
+const { table } = require('table');
+const { DefinePlugin } = require('webpack');
+const { ProvidePlugin } = require('webpack');
```

Comment:
I think you can destructure `DefinePlugin` and `ProvidePlugin` on the same line:

`const { DefinePlugin, ProvidePlugin} = require('webpack');`

Reply:
Good idea! Put this change in.